### ISSUE TITLE
Update called two times for nested view

### ISSUE THREAD

**Issue by activey** [yohamta/furex#71]
Hi,

I've just found my View is Updated two times when it's a child of another view, like in this setup:

```go

type TestHandler struct {
}

func (t TestHandler) Update(v *furex.View) {
  if inpututil.IsKeyJustPressed(ebiten.KeyQ) {
    fmt.Println("XXX")
  }
}

rootView := &furex.View{
}

nestedView := &furex.View{
  Handler:      &TestHandler{},
}

rootView.AddChild(nestedView)

```

and now whenever I call press Q then XXX is printed two times, it does not happen when Handler is set in rootView instead. 

Many thanks for any responses :)
BR

**Comment by activey** [comment]
I have theory:

```go
// Update updates the view
func (v *View) Update() {
	if v.isDirty {
		v.startLayout()
	}
	v.processHandler()
	for _, v := range v.children {
		v.item.Update()
		v.item.processHandler()
	}
	if !v.hasParent {
		v.processEvent()
	}
}
```
I have a feeling the first v.processHadler() should be called only when v has no parent...

**Comment by yohamta** [comment]
@activey Good catch, thank you! The fix also seems fine to me :)

**Comment by yottahmd** [comment]
This issue is being closed because it has been addressed in #72.
